### ISSUE TITLE
does this library work client side (in browser)?

### ISSUE THREAD

**Issue by chovy** [simov/slugify#48]
Please add instructions for loading from cdnjs or something similar.

**Comment by simov** [comment]
Yes, it works. Documentation would be helpful. Here is the link: https://unpkg.com/slugify@1.3.4/index.js the global object is called `slugify`.
